src/software/jetson_nano/services/motor.h

```diff
@@ -123,6 +128,75 @@ class MotorService
         }
     };
 
+    /**
+     * Log the driver fault in a human readable log msg
+     *
+     * @param motor The motor to log the status for
+     *
+     * @return a struct containing the motor faults and whether the motor was disabled due
+     * to the fault
+     */
+    struct MotorFaultIndicator checkDriverFault(uint8_t motor);
```

Comment:
Is there a reason why this strict is being moved from private to public?

Reply:
I won't be able to implement checkDriverFault if the struct is private.
![image](https://github.com/UBC-Thunderbots/Software/assets/47040320/23e647dd-0dae-475c-b971-32df72a141c8)